apps/coscrad-frontend/src/store/slices/resources/playlists/thunks/fetchPlaylists.ts

```diff
@@ -0,0 +1,10 @@
+import { buildResourceFetchActionPrefix } from '../../../utils/build-resource-fetch-action-prefix';
+import { createFetchThunk } from '../../../utils/create-fetch-thunk';
+import { getApiResourcesBaseRoute } from '../../shared';
+import { PLAYLISTS } from '../constants';
+import { PlaylistIndexState } from '../types';
+
+export const fecthPlaylists = createFetchThunk<PlaylistIndexState>(
+    buildResourceFetchActionPrefix(PLAYLISTS),
+    `${getApiResourcesBaseRoute()}/Playlists`
```

Comment:
Note that the casing here is inconsistent with the previous one. You need to make sure that the link is actually working.